src/testing/vortex/workload.zig

```diff
@@ -0,0 +1,407 @@
+//! This workload runs in an loop, generating and executing operations on a cluster.
+//!
+//! Any sucessful operations are reconciled with a model, tracking what accounts exist. Future
+//! operations are generated based on this model.
+//!
+//! After every operation, all accounts are queried, and basic invariants are checked.
+
+const std = @import("std");
+const Driver = @import("./driver.zig");
+const arbitrary = @import("./arbitrary.zig");
+const tb = @import("../../tigerbeetle.zig");
+
+const log = std.log.scoped(.workload);
+const assert = std.debug.assert;
+const testing = std.testing;
+
+const events_count_max = Driver.events_count_max;
+const pending_transfers_count_max = 1024;
+const accounts_count_max = 128;
+
+pub const CLIArgs = struct {
+    @"cluster-id": u128,
+    addresses: []const u8,
+    @"driver-command": []const u8,
+};
+
+pub fn main(allocator: std.mem.Allocator, args: CLIArgs) !void {
```

Comment:
Let's make the interface more crisp here. Instead of workload spawning the driver, let's just pass in driver-s pipes:

```
fn main(
  allocator, 
  args, 
  pipes: struct {in : std.fs.File, out: std.fs.File }

)
```

and let the superviser to spawn the driver.